README.rst

```diff
@@ -28,12 +28,22 @@ How can I contribute?
 What do I need to contribute?
 -----------------------------
 
-* A host with python 3.6+, gcc and the python development header files
-* tox
-* go
+* A host with `python` 3.6+ and `tox`
 * docker and/or podman+buildah
 * vagrant (optional, can be used to test FIPS mode and registered hosts)
 
+Test setup on openSUSE
+^^^^^^^^^^^^^^^^^^^^^^
+
+To setup the test environment on openSUSE Leap 15.6 and Tumbleweed, run the following commands:
+
+
+.. code-block:: shell-session
+
+    $ sudo zypper -n in podman buildah docker git-core python3 python3-pip
+    $ pip3 --quiet install --upgrade pip
+    $ pip3 --quiet install tox --ignore-installed six
```

Comment:
Maybe add a `--user` flag here?
```suggestion
    $ pip3 --quiet install --user --upgrade pip
    $ pip3 --quiet install --user tox --ignore-installed six
```

Reply:
I think we should advertise installation of packages, or if that is unwanted for some reason, advertise the use of venvs instead of pip install globally (or as user which is also not possible on tumbleweed)

Reply:
I do like to advertise the use of venvs more than using the `--user` flag, so I added instructions to use a virtualenv.

By using the `--user` flag we need to explain that this will only work outside of virtualenvs and it's also not what we are using in our test flow in openQA, so I think that's not what we should document.

Reply:
actually I'd prefer if we wouldn't use the pip method in openQA either from a supply chain security perspective..

Reply:
Let's make this a meeting point on Tuesday, that requires more more engineering work especially on older versions (12-SP5) but if we want to do this, it should be possible.

For now I suggest we decouple this initiative from this PR, because that's not easy to solve and not worth waiting for IMHO.